src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.IncludeElementExpander.cs

```diff
@@ -576,23 +579,27 @@ private static Binder MakeNameBinder(bool isParameter, Symbol memberSymbol, CSha
                 }
                 else
                 {
-                    switch (memberSymbol.Kind)
+                    bool continueSearch = true;
+                    for (var currentSymbol = memberSymbol; currentSymbol != null && continueSearch; currentSymbol = currentSymbol.ContainingSymbol, continueSearch = isTypeRef)
```

Comment:
>continueSearch = isTypeRef [](start = 148, length = 26)

I didn't fully understand this. Could the loop be written differently (maybe a do/while) to make the intent clearer?
In particular `continueSearch = isTypeRef` was strange. Maybe it would be clearer to have a flag for "first iteration" (if needed).